qaframework-bdd-tests/src/test/resources/features/refapp-2.x/05-location/locations.feature

```diff
@@ -1 +1,28 @@
-Feature: Location Management
\ No newline at end of file
+Feature: Location Management
+
+  Background:
+    Given a user clicks on the configure metadata link from the home page
+
+
+  @selenium
+  @location
+  Scenario: LOCATION_MANAGEMENT
+    # User story: Add Location Attribute type
+    When the system loads configure metadata link dashboard
+    And a user clicks on Manage Location Attribute Types
```

Comment:
don't you agree with me that when `a user clicks on Manage Location Attribute Types`, then `the system loads the manage location attribute types page`?